ms_gsl.sh

```diff
@@ -0,0 +1,31 @@
+package: ms_gsl
+version: master
```

Comment:
Please fix the linting and pick an actual tag / commit. We should not depend on master.

Reply:
fixed